utils/rand_kv.go

```diff
@@ -3,28 +3,24 @@ package utils
 import (
 	"fmt"
 	"math/rand"
-	"sync"
 	"time"
 )
 
 var (
-	lock    = sync.Mutex{}
 	randStr = rand.New(rand.NewSource(time.Now().Unix()))
 	letters = []byte("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
 )
 
-// GetTestKey 获取测试使用的 key
+// GetTestKey get formated key, for test only
 func GetTestKey(i int) []byte {
 	return []byte(fmt.Sprintf("rosedb-test-key-%09d", i))
 }
 
-// RandomValue 生成随机 value，用于测试
+// RandomValue generate random value, for test only
 func RandomValue(n int) []byte {
 	b := make([]byte, n)
 	for i := range b {
-		lock.Lock()
```

Comment:
We should not remove this lock, otherwise there will be concurrency issues for some test cases.

Reply:
This lock is to protect the rand object, and it really needs to be locked. 😁